qiskit/unroll/_jsonbackend.py

```diff
@@ -270,6 +277,9 @@ def start_gate(self, name, args, qubits, nested_scope=None):
             self.listen = False
             qubit_indices = [self._qubit_order_internal.get(qubit)
                              for qubit in qubits]
+            # Check for special simulator commands.
+            if name in SIMULATION_COMMANDS:
+                name = '#%s' % name
```

Comment:
Yes, I'd like to standardize the string formatting that we use at some point ('%' and 'format()' are both used through the codebase), and actually favoring the `.format()` one. Since it's non blocking, I'll leave it to a separate PR (might be a `good first contributor` one, actually!).